sections/header.liquid

```diff
@@ -123,7 +123,7 @@
   </symbol>
 </svg>
 <{% if section.settings.enable_sticky_header %}sticky-header{% else %}div{% endif %} class="header-wrapper color-{{ section.settings.color_scheme }} gradient{% if section.settings.show_line_separator %} header-wrapper--border-bottom{% endif %}">
-  <header class="header header--{{ section.settings.logo_position }} page-width{% if section.settings.menu != blank %} header--has-menu{% endif %}">
+  <header class="header header--{{ section.settings.logo_position }} header--mobile-{{ section.settings.mobile_logo_position }} page-width{% if section.settings.menu != blank %} header--has-menu{% endif %}">
```

Comment:
```suggestion
  <header class="header header--{{ section.settings.logo_position }} header-mobile--{{ section.settings.mobile_logo_position }} page-width{% if section.settings.menu != blank %} header--has-menu{% endif %}">
```

Because the position is a modifier, does this make more sense? 🤔

Reply:
I personally think because `header` is the element, a `--mobile-center` makes more sense than adding a `header-mobile` class with a `--center` modifier 👀

Would love to get other reviewer's thoughts on this.

Reply:
Fair point 🤔 My only thought is that we have

header--{{ section.settings.logo_position }} 
and
header--mobile-{{ section.settings.mobile_logo_position }}

Where the modifiers should both exist so they are not the same modifier
But they are both modifiers on `header` which I find confusing since they should both exist

Does that make sense?

Reply:
I don't have a very strong opinion on this but I'll share my thoughts ❤️ 

My suggestion would be to update the `mobile_logo_position` setting values to `mobile-center` and `mobile-left` then the above would simply become `header--{{ section.settings.mobile_logo_position }}`.

Another option is to add `-mobile` after the modifier: `header--{{ section.settings.mobile_logo_position }}-mobile`. And to Sofia's point, I think the confusion potential is there but at the same time, when I first looked at this, I didn't find it confusing but I'm often in favour in improving readability where we can:

<img src="https://user-images.githubusercontent.com/28404165/180877363-05680d71-3f13-4938-813f-2daaaf7b6605.png" width="1000" height="auto">

Reply:
I think I would keep header since `mobile-center` would be more of a generic class and not specific to this file. 🤔 Thoughts?

We can keep it like how it is, but thought id share these thoughts 😅

Reply:
@LucasLacerdaUX I dont have any strong feelings either way, but thought i'd bring it up :)

thank you for explaining it 🙌

Reply:
Thanks for sharing your thoughts, Lucas!

Sorry for the can of 🪱's 😅 

Lucas, since we now have a good discussion around this topic in this thread + since both Sofia and I don't have very strong feelings, I think you can leave it as is 🙌🏼